test/plugins/test_convert.py

```diff
@@ -347,3 +347,59 @@ def test_transcode_from_lossy_prevented(self):
             self.run_convert_path(item.path)
         converted = os.path.join(self.convert_dest, b"converted.ogg")
         self.assertNoFileTag(converted, "mp3")
+
+
+@_common.slow_test()
+class NoConvertTest(ConvertTestCase, ConvertCommand):
+    """Test the effect of the `no_convert` option."""
+
+    def setUp(self):
+        super().setUp()
+
+        self.convert_dest = os.path.join(self.temp_dir, b"convert_dest")
+        self.config["convert"] = {
+            "dest": self.convert_dest,
+            "no_convert": "format:OGG",
+            "paths": {"default": "converted"},
+            "format": "mp3",
+            "formats": {
+                "mp3": self.tagged_copy_cmd("mp3"),
+            },
+        }
+
+    def test_no_transcode_when_no_convert_set(self):
+        [item] = self.add_item_fixtures(ext="ogg")
+        with control_stdin("y"):
+            self.run_convert_path(item.path)
+        converted = os.path.join(self.convert_dest, b"converted.mp3")
+        self.assertFalse(os.path.exists(converted))
```

Comment:
Once you rebase off/merge `master`, you will find that `unittest`-specific assertions have been replaced with `assert != ...` statements as we're migrating to `pytest`. SO it would be good to update your tests accordingly

Reply:
I have updated these assertions